include/openthread/platform/radio.h

```diff
@@ -1142,6 +1142,20 @@ otError otPlatRadioEnableCsl(otInstance         *aInstance,
                              otShortAddress      aShortAddr,
                              const otExtAddress *aExtAddr);
 
+/**
+ * Reset CSL receiver in the platform.
+ *
+ * @note Defaults to `otPlatRadioEnableCsl(aInstance,0, Mac::kShortAddrInvalid, nullptr);`
+ *
+ * @param[in]  aInstance     The OpenThread instance structure.
+ *
+ * @retval  kErrorNotImplemented Radio driver doesn't support CSL.
+ * @retval  kErrorFailed         Other platform specific errors.
+ * @retval  kErrorNone           Successfully disabled CSL.
```

Comment:
These should be `OT_ERROR_*`

Reply:
fixed for enable csl implementation as well